### ISSUE TITLE
Sustainfest(?), launching a new landing page and campaign for getting more contributors involved

### ISSUE THREAD

**Issue by Codeshark-NET** [protontypes/open-sustainable-technology#217]
**As an organisation, we would like to get contributors more involved with our proposed projects, so that our community can grow and we can facilitate real-life impact & growth.**

The suggestion here is to create a landing-page, detailing issues that will be good for first-time contributors to a given project, along with a guide on how to get involved in said projects.

The issues should be filterable by "language" and "category".

The page should also hold a bit of a "manifesto" or "guide".

----

**Meaning we need 4 things:**
* A backend/api for delivering the issues
* A frontend for showing this information / pulling the issues
* A marketing campaign for pushing this out there, both with a strategy for social media and with our partners
* A method of counting/tracking contributions via this campaign (perhaps via #hastag in the pr/issues)

As discussed with @Ly0n and @andrew. 🙌

**Comment by Codeshark-NET** [comment]
I'm happy to take the lead on pushing that out there, and will be starting to work on the landingpage in the next days. 👍

**Comment by RichardLitt** [comment]
Small note: Sustainfest might be confusing. Open Sustainable Tech Fest, maybe?

**Comment by Codeshark-NET** [comment]
> Small note: Sustainfest might be confusing. Open Sustainable Tech Fest, maybe?

Absolutely, several aspects of the naming and campaign still need to be determined. Perhaps there's some way we could track that here too, although it is a bit marketing focused, what do you reckon?

**Comment by RichardLitt** [comment]
Marketing is how your project becomes known - it's a core part of the process. 

One thing I would say - I'm not sure setting up a mechanishm for first time contributors outside of the project is as useful as showcasing the value of the project to contributors through the main access points. What's in it for contributors? That question is more important than surfacing small issues that they may or may not be able to do.

**Comment by Ly0n** [comment]
> Small note: Sustainfest might be confusing. Open Sustainable Tech Fest, maybe?

We should not name it SustainFest, as this would lead to confusion with SustainOSS. Other option could be OpenSustainFest or OpenSustainabilityFest. We could also remove the Fest completly and call it OpenSustainMatch :partying_face: 

> One thing I would say - I'm not sure setting up a mechanishm for first time contributors outside of the project is as useful as showcasing the value of the project to contributors through the main access points. What's in it for contributors? That question is more important than surfacing small issues that they may or may not be able to do.

Maybe some background here. What comes a little bit short in the description, is that we would not like to highlight the issues of this projects but from all project listed on OpenSustain.tech. Many people contacted me in the past, looking for open source project in the sustainability sector they can contribute to. What we are planning to do is to match projects with people. @andrew already did most of the backend work: https://ost.ecosyste.ms/issues
This page just showes the open issues from OpenSustain.tech that are labeled with firstgoodissue, hacktoberfest and help wanted. The current database is just to hard for people to browse and they get lost in to many projects. Also most of the projects demand a lot of domain expertise. With this page we can help guide people to projects that are beginnner friendly but still targeting a sustainability goal.

In a first iteration we would like people to be able to filter issues by project domain ( like Photovolatic) and programming languages (Scala). 

What @RichardLitt is right about, is the fact that we maybe should not show people the issue directly but the projects that are having this labels.

> What's in it for contributors? 

We do have some ideas for OpenSustain.tech. One is already in the issues #199.  The other will be that contributors can create a note on the website landing page on a prominent position.
